packages/auth/e2e/user.e2e.js

```diff
@@ -174,6 +174,35 @@ describe('auth().currentUser', () => {
 
       return Promise.resolve();
     });
+
+    it('should work with actionCodeSettings', async () => {
+      const actionCodeSettings = {
+        handleCodeInApp: true,
+        url: 'https://react-native-firebase-testing.firebaseapp.com/foo',
+      };
+      await Utils.sleep(3000);
+      const random = Utils.randString(12, '#aA');
+      const email = `${random}@${random}.com`;
+      await firebase.auth().createUserWithEmailAndPassword(email, random);
+
+      try {
+        await firebase.auth().currentUser.sendEmailVerification(actionCodeSettings);
+        await firebase.auth().currentUser.delete();
+      } catch (error) {
+        console.log(error);
```

Comment:
```suggestion
        console.log(error);
```

already a log on line 200